hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -54,11 +53,6 @@ protected ServiceName getParentServiceName(PathAddress parentAddress) {
     }
 
     @Override
-    protected boolean isResourceServiceRestartAllowed(OperationContext context, ServiceController<?> service) {
```

Comment:
I just would like to note that if we don't allow a restart (i.e. `isResourceServiceRestartAllowed` returns `false`) the whole server will be put in `reload required´ state. See [here](https://github.com/wildfly/wildfly-core/blob/master/controller/src/main/java/org/jboss/as/controller/RestartParentWriteAttributeHandler.java#L77-L78) and [here](https://github.com/wildfly/wildfly-core/blob/master/controller/src/main/java/org/jboss/as/controller/AbstractWriteAttributeHandler.java#L155-L156) for more info.

This can be avoided by always returning false on `applyUpdateToRuntime`, but then, we wouldn't know if the agent needs a restart.